diagram-viewer/src/main/resources/sld/view.fxml

```diff
@@ -64,8 +64,8 @@
                         <Label text="Substation Layout:"/>
                         <ComboBox fx:id="substationLayoutComboBox"/>
                         <Label text="VoltageLevel Layout:"/>
-                        <ComboBox fx:id="voltageLevelLayoutComboBox"/>
-                        <CheckBox fx:id="stackFeedersCheckBox" text="Stack feeders"/>
+                        <EnumChoiceBox enumType="com.powsybl.diagram.viewer.sld.SingleLineDiagramModel$VoltageLevelLayoutFactoryType" fx:id="voltageLevelLayoutComboBox" initialValue="SMART"/>
+                        <CheckBox fx:id="stackFeedersCheckBox" text="Stack feeders" selected="true"/>
```

Comment:
there's also removeUnnecessaryFictitiousNodes and substituteSingularFictitiousByFeederNode which are supposed to be true by default